### PR TITLE
Fei's completed deliverable (Homework 3)

### PR DESCRIPTION
**Pull request by f-eiyu**
**Deliverable Submission**

Please describe your comfort and completeness levels before submitting.

Comfort Level (1-5): 5

Completeness Level (1-5): 5

What did you think of this deliverable?: straightforward and snappy, a nice exercise!

### CODE REVIEW

File: js/pluralizer.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+// Your program should pluralize the word based on an integer (count)
+
+const thing = "integral";
+const count = 1;
+
+// this is actually a wildly complicated task, but for my own well-being i'll assume that
+// you're not trying to ask about the absolute linguistic disaster of english plural forms
+// and instead are going for a nice, well-behaved coding exercise
+console.log(thing + (count > 1 ? "s" : ""));
```
Comment (kestler01) [inline js/pluralizer.js:18]: what if i have 0 dogs ?

**Comment by kestler01** [comment]
love the commit history! very well done ✅

File: js/pluralizer.js (hunk 1 above)
Comment (f-eiyu) [reply js/pluralizer.js:18]: i genuinely wasn't even thinking hard enough for this problem to want to deal with this special case LMAO i promise i'll actually error handle properly in the projects and future assignments :pray:

Comment (kestler01) [reply js/pluralizer.js:18]: no worries ! just trying to push you a little and get you to think about edge cases 😁

**Pull request closed without merging**